website/src/resources/events.json

```diff
@@ -1,31 +1 @@
-[
-    {
```

Comment:
Can we add event format as a comment?

Reply:
or may be adding date format in `interface` will help??

Reply:
>or may be adding date format in interface will help??

string will be good enough right! i have no idea on date interface, if you can give me some idea...

Reply:
`Date` type

Reply:
We need to use DateTime reviver in JSON.parse to be able to use `Date` in the interface.  Anyway, we can go with the string also as long as the string format is consistent . Approving this PR.